webapp/channels/src/sass/responsive/_mobile.scss

```diff
@@ -10,20 +10,15 @@
             }
 
             .navbar-header {
-                display: flex;
-                align-items: center;
-                gap: 4px;
```

Comment:
The new CSS for this is definitely nicer, but I reverted it because this `gap` messed with the spacing. One of the things that it did was that It applied unexpectedly to the space before the first item in the `navbar-header` which wouldn't normally happen, but it seems to happen because Bootstrap 3 adds a `::before` element to this as a clearfix and `gap` adds space between a pseudoelement like that and the first element apparently